WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -553,13 +553,13 @@ class PostListViewController : AbstractPostListViewController, UIViewControllerR
 
     // MARK: - Filter Related
 
-    override func currentPostAuthorFilter() -> PostAuthorFilter {
-        if !canFilterByAuthor() {
+    override class func currentPostAuthorFilter(blog blog:Blog) -> PostAuthorFilter {
```

Comment:
Kind of the same comment as above... having a class getter and an instance setter doesn't make much sense.  Can we unify them?